### PR TITLE
Every error will throw an exception on our test suites

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | We try to catch and fix every "silent" error we could have in PrestaShop. A first strategy could be to throw an exception in tests 
| Type?         | bug fix
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Part of https://github.com/PrestaShop/PrestaShop/projects/1#card-13082162
| How to test?  | I expect this pull request to break everything, but once I'll fix the world, the world will be a better world :sunglasses: 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10472)

### CODE REVIEW

**Comment by matks** [comment]
💥

File: tests/phpunit.xml [hunk 1]
```diff
@@ -1,5 +1,10 @@
-<phpunit bootstrap="bootstrap.php"
-         color="true">
+<phpunit
+    bootstrap="bootstrap.php"
+    color="true"
+    convertErrorsToExceptions   = "true"
```
Comment (PierreRambaud) [inline tests/phpunit.xml:4]: Why not the same spaces as color and bootstrap? :D

Comment (mickaelandrieu) [reply tests/phpunit.xml:4]: because it's a copy paste? ^^

**Comment by mickaelandrieu** [comment]
hmmm I can tell for sure that the test suite launch errors, so the PHPUnit configuration doesn't work as expected, see https://github.com/PrestaShop/PrestaShop/pull/10477

**Comment by mickaelandrieu** [comment]
closed in favor of #10477

**Pull request closed without merging**